shrinkwrap/elf.py

```diff
@@ -40,7 +40,7 @@ def explore(self, binary: lief.Binary, filename: str) -> Dict[str, str]:
         # TODO: Figure out why `--list` and `ldd` produce different outcomes
         # specifically for the interpreter.
         # https://gist.github.com/fzakaria/3dc42a039401598d8e0fdbc57f5e7eae
-        for line in resolution:
+        for line in resolution.splitlines():
```

Comment:
Okay I figured it out -- this isn't actually a String.
if you look above it's the output of a Command object using the library
https://amoffat.github.io/sh/index.html

The result of the command is the type _ResultCommand_ which behaves very much like a string.
Looks like when it's iterated it already returns line by line results.

Reply:
I'll add a comment explaining.